sharpy/postproc/frequencyresponse.py

```diff
@@ -137,8 +137,8 @@ def run(self):
             else:
                 compute_fom = True
 
-        cout.cout_wrap('Computing frequency response...')
         if (self.settings['compute_fom'].value and self.settings['load_fom'] == '') or compute_fom:
+            cout.cout_wrap('Computing frequency response...')
```

Comment:
Shouldn't the `cout` stuff be wrapped in a `if self.settings['print_info']`?

Reply:
Indeed, preparing a new commit